### PR TITLE
[WIP] chromium: update to 72.0.3626.119.

### PR DESCRIPTION
**Pull request by Cogitri**
[ci skip]

### CODE REVIEW

File: srcpkgs/chromium-widevine/template [hunk 1]
```diff
@@ -3,18 +3,18 @@ _chromeVersion="current"
...
 license="chrome"
-only_for_archs="x86_64"
 depends="chromium binutils xz"
+homepage="https://www.google.com/chrome"
 repository=nonfree
-create_wrksrc=yes
 distfiles="https://dl.google.com/linux/direct/google-chrome-${_channel}_${_chromeVersion}_amd64.deb"
-checksum=c8bf514f3c70868120924ee2d4872a0e47dcff315bdf799ba2f98dd7aa85fe2b
+checksum=4513f80f2e858b85d321cd10f7a7cddb3a3aa9f6ac4e903dc0d52493337e3a13
+archs="x86_64"
```
Comment (maxice8) [inline srcpkgs/chromium-widevine/template]: archs should be where noarch=yes would be

File: srcpkgs/chromium/template [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 revision=1
 short_desc="Google's attempt at creating a safer, faster, and more stable browser"
 maintainer="Rasmus Thomsen <rasmus.thomsen@protonmail.com>"
 license="BSD-3-Clause"
 homepage="https://www.chromium.org/"
 distfiles="https://commondatastorage.googleapis.com/chromium-browser-official/${pkgname}-${version}.tar.xz"
-checksum=574a5400dcd9452bf41481518aa5097c396df6e1eb97c67303638ba967048c2d
+checksum=b3953ce18044117d98f0b1b9922aeb93030a7b49b85b140060c6fe3e467a917a
 
 archs="i686 x86_64 x86_64-musl"
```
Comment (maxice8) [inline srcpkgs/chromium/template]: please move it to where noarch=yes would be

File: srcpkgs/chromium-widevine/template (hunk 1 above)
Comment (Cogitri) [reply srcpkgs/chromium-widevine/template]: Is it just me, or does xlint not find this?

**Comment by maxice8** [comment]
On Sat, Feb 23, 2019 at 02:15:12PM -0800, Rasmus Thomsen wrote:
> Cogitri commented on this pull request.
> 
> 
> 
> >  distfiles="https://dl.google.com/linux/direct/google-chrome-${_channel}_${_chromeVersion}_amd64.deb"
> -checksum=c8bf514f3c70868120924ee2d4872a0e47dcff315bdf799ba2f98dd7aa85fe2b
> +checksum=4513f80f2e858b85d321cd10f7a7cddb3a3aa9f6ac4e903dc0d52493337e3a13
> +archs="x86_64"
> 
> Is it just me, or does xlint not find this?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/9082#discussion_r259595210

using last git ?

**Comment by Cogitri** [comment]
Yup

File: srcpkgs/chromium-widevine/template (hunk 1 above)
Comment (maxice8) [reply srcpkgs/chromium-widevine/template]: only git xlint finds it

**Pull request merged**